### PR TITLE
test: added ids for testing

### PR DESCRIPTION
**Pull request by DimitrySakalouski**

### CODE REVIEW

File: src/shared/ui/survey/CheckBox/CheckBox.item.tsx [hunk 1]
```diff
@@ -90,6 +90,7 @@ const CheckBoxItem: FC<Props> = ({
 
       <CheckBox
         style={styles.checkbox}
+        data-test={`checkbox-${value}`}
```
Comment (anq83) [inline src/shared/ui/survey/CheckBox/CheckBox.item.tsx:93]: Isn't value true or false here? If so, it cannot be part of test attribute

File: src/shared/ui/survey/RadioActivityItem/RadioActivityItem.tsx [hunk 2]
```diff
@@ -47,6 +47,7 @@ const RadioActivityItem: FC<RadioActivityItemProps> = ({
         value={initialValue?.id ?? ''}
         onValueChange={onValueChange}
         name="radio"
+        data-test="radio-item-group"
       >
         {optionsList.map(option => (
           <Box my="$1" key={option.id} onPress={() => onValueChange(option.id)}>
```
Comment (anq83) [inline src/shared/ui/survey/RadioActivityItem/RadioActivityItem.tsx:53]: Shouldn't we add test attribute into each option?

File: src/shared/ui/survey/RadioActivityItem/RadioItem.tsx [hunk 3]
```diff
@@ -83,20 +86,26 @@ const RadioItem: FC<RadioLabelProps> = ({
         <Box width="10%">
           <CachedImage
             resizeMode="contain"
+            data-test={`radio-option-image-${value}`}
```
Comment (anq83) [inline src/shared/ui/survey/RadioActivityItem/RadioItem.tsx:89]: What is the value here? True / false or text ?

File: src/widgets/activity-group/ui/ActivityGroup.tsx [hunk 4]
```diff
@@ -78,9 +78,18 @@ const ActivityGroup: FC<Props> = ({ appletId, group, ...styledProps }) => {
   };
 
   return (
-    <Box {...styledProps}>
+    <Box
+      data-test={`activity-group-${group.type}-${group.name}`}
```
Comment (anq83) [inline src/widgets/activity-group/ui/ActivityGroup.tsx:82]: group.name is key for i18, no need to use it, group.type is enough

**Review by anq83 — CHANGES_REQUESTED** [review]

File: src/shared/ui/survey/RadioActivityItem/RadioItem.tsx (hunk 3 above)
Comment (DimitrySakalouski) [reply src/shared/ui/survey/RadioActivityItem/RadioItem.tsx:89]: value is number - unique for each option in radio group

**Review by anq83 — APPROVED** [review]

**Pull request merged**